### PR TITLE
feat(com-pwa): new sign-in page and auth process

### PR DESCRIPTION
**Pull request by njfamirm**

### CODE REVIEW

File: core/type/src/customer-order-management.ts [hunk 1]
```diff
@@ -142,6 +144,16 @@ export interface OrderShippingInfo extends StringifyableRecord {
...
 }
 
+export interface ComUser extends User {
+  shopName: string;
+  tel: string;
+  province: string;
+  city: string;
+  address: string;
+  postalCode: string;
+  description?: string;
```
Comment (AliMD) [inline core/type/src/customer-order-management.ts:154]: .

File: core/type/src/customer-order-management.ts [hunk 2]
```diff
@@ -142,6 +144,16 @@ export interface OrderShippingInfo extends StringifyableRecord {
   timePeriod: (typeof timePeriodCS)[number];
 }
 
+export interface ComUser extends User {
+  shopName: string;
```
Comment (AliMD) [inline core/type/src/customer-order-management.ts:148]: opt

File: core/type/src/customer-order-management.ts [hunk 3]
```diff
@@ -142,6 +144,16 @@ export interface OrderShippingInfo extends StringifyableRecord {
...
+  tel: string;
```
Comment (AliMD) [inline core/type/src/customer-order-management.ts:149]: Landline phone

File: uniquely/com-pwa/src/manager/context-provider/user.ts [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+import {config} from '../../config.js';
+
+import type {AlwatrDocumentStorage} from '@alwatr/type';
+import type {ComUser} from '@alwatr/type/customer-order-management.js';
+
+export const userStorageContextConsumer = serverContextConsumer<AlwatrDocumentStorage<ComUser>>(
+    'user_storage_context',
+    {
+      ...config.fetchContextOptions,
+      url: config.authApi,
```
Comment (AliMD) [inline uniquely/com-pwa/src/manager/context-provider/user.ts:12]: .

File: core/type/src/customer-order-management.ts [hunk 5]
```diff
@@ -142,6 +144,15 @@ export interface OrderShippingInfo extends StringifyableRecord {
   timePeriod: (typeof timePeriodCS)[number];
 }
 
+export interface ComUser extends User {
+  landlinePhone: string;
+  province: string;
+  city: string;
+  address: string;
+  postalCode: string;
+  shopName?: string;
```
Comment (njfamirm) [inline core/type/src/customer-order-management.ts:153]: optional?

File: uniquely/com-pwa/src/content/l18e-fa.json [hunk 6]
```diff
@@ -42,6 +42,8 @@
 
     "page_order_tracking_headline": "پیگیری سفارش",
 
+    "page_sing_in_headline": "ورود",
```
Comment (njfamirm) [inline uniquely/com-pwa/src/content/l18e-fa.json:45]: sign

File: uniquely/com-pwa/res/config.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
 window.appConfig = {
   api: 'https://order.soffit.co/api/v0',
-  cdn: 'https://order.soffit.co/cdn/',
+  cdn: 'https://order.soffit.co/cdn',
```
Comment (njfamirm) [inline uniquely/com-pwa/res/config.js:3]: version?

File: uniquely/com-pwa/src/manager/buttons.ts [hunk 8]
```diff
@@ -94,6 +94,11 @@ export const buttons = {
     icon: 'reload-outline',
     clickSignalId: 'retry_click_event',
   },
+  singIn: {
```
Comment (njfamirm) [inline uniquely/com-pwa/src/manager/buttons.ts:97]: sign

File: uniquely/com-pwa/src/ui/alwatr-pwa.ts [hunk 9]
```diff
@@ -1,14 +1,14 @@
-import {html, customElement} from '@alwatr/element';
+import {html, customElement, nothing} from '@alwatr/element';
 import '@alwatr/font/vazirmatn.css';
 import {AlwatrPwaElement} from '@alwatr/pwa-helper/pwa-element.js';
+import {RouteContext, RoutesConfig, redirect} from '@alwatr/router';
```
Comment (njfamirm) [inline uniquely/com-pwa/src/ui/alwatr-pwa.ts:4]: import type

File: uniquely/com-pwa/src/ui/page/sign-in.ts [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+
+import {buttons} from '../../manager/buttons.js';
+import {signIn, userStorageContextConsumer} from '../../manager/context-provider/user.js';
+import {topAppBarContextProvider} from '../../manager/context.js';
+
+import type {AlwatrTextField} from '@alwatr/ui-kit/text-field/text-field.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'alwatr-page-login': AlwatrPageSignIn;
```
Comment (njfamirm) [inline uniquely/com-pwa/src/ui/page/sign-in.ts:27]: sign-in

**Review by AliMD — APPROVED** [review]

**Pull request merged**